docker-compose.yml

```diff
@@ -26,3 +26,11 @@ services:
     ports:
       - "55681:55681"
 
+  profile_mock_server:
+    image: stoplight/prism:4
+    command: -p 29712 mock --host 0.0.0.0 "/etc/openapi.yml"
```

Comment:
I use the port 29712 here just for the ease of logs reading because `docke-compose up` will now print this:
```
profile_mock_server_1      | [12:17:59 PM] › [CLI] …  awaiting  Starting Prism…
dynamodb-local             | 2021-04-22 12:18:00.029:INFO::main: Logging initialized @1678ms to org.eclipse.jetty.util.log.StdErrLog
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      POST       http://0.0.0.0:29712/users
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      GET        http://0.0.0.0:29712/users/et
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      PATCH      http://0.0.0.0:29712/users/aut
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      GET        http://0.0.0.0:29712/users/me
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      PATCH      http://0.0.0.0:29712/users/me
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      GET        http://0.0.0.0:29712/users/me/addresses
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      POST       http://0.0.0.0:29712/users/me/addresses
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      GET        http://0.0.0.0:29712/users/me/profile
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      POST       http://0.0.0.0:29712/users/me/profile
profile_mock_server_1      | [12:18:00 PM] › [CLI] ℹ  info      PATCH      http://0.0.0.0:29712/users/me/profile
profile_mock_server_1      | [12:18:00 PM] › [CLI] ▶  start     Prism is listening on http://0.0.0.0:29712
```